torch/distributed/_fsdp/utils.py

```diff
@@ -0,0 +1,48 @@
+from typing import Dict, List, Tuple, Union, Any, Callable, Set
+from collections import OrderedDict  # noqa: F401
+
+import torch
+from torch.nn.utils.rnn import PackedSequence
+import torch.nn.functional as F
+
+
+"""Useful functions to deal with tensor types with other python container types."""
+
+
+def apply_to_tensors_(fn: Callable, container: Union[torch.Tensor, Dict, List, Tuple, Set]) -> Any:
```

Comment:
ditto

Besides, these two APIs seems are generic APIs. @albanD do we have similar APIs in other parts of PyTorch?